src/pages/upload/index.tsx

```diff
@@ -371,4 +383,134 @@ const Upload: React.FC<
   )
 }
 
+const SubmitStatus = ({submitResponse}: {submitResponse: SubmitResponse}) => {
+  if (!submitResponse) return null
+
+  if (submitResponse.status === 200) {
+    const studioUrl =
+      process.env.NODE_ENV === 'production'
+        ? 'https://egghead-next.sanity.studio/desk'
+        : 'https://egghead-next-test.sanity.studio/desk'
+
+    return (
+      <div className="rounded-md bg-green-50 p-4">
+        <div className="flex">
+          <div className="flex-shrink-0">
+            <svg
+              className="h-5 w-5 text-green-400"
+              xmlns="http://www.w3.org/2000/svg"
+              viewBox="0 0 20 20"
+              fill="currentColor"
+              aria-hidden="true"
+            >
+              <path
+                fill-rule="evenodd"
+                d="M10 18a8 8 0 100-16 8 8 0 000 16zm3.707-9.293a1 1 0 00-1.414-1.414L9 10.586 7.707 9.293a1 1 0 00-1.414 1.414l2 2a1 1 0 001.414 0l4-4z"
+                clip-rule="evenodd"
+              />
+            </svg>
+          </div>
+          <div className="ml-3">
+            <h3 className="text-sm font-medium text-green-800">
+              Course Submitted
+            </h3>
+            <div className="mt-2 text-sm text-green-700">
+              <p>
+                The Course and Lesson metadata has been successfully submitted
+                to Sanity Studio
+              </p>
+            </div>
+            <div className="mt-4">
+              <div className="-mx-2 -my-1.5 flex">
+                <a
+                  href={studioUrl}
+                  className="bg-green-50 px-2 py-1.5 rounded-md text-sm font-medium text-green-800 hover:bg-green-100 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-offset-green-50 focus:ring-green-600"
+                >
+                  View Studio
+                </a>
+              </div>
```

Comment:
Link to the environment-appropriate Sanity studio after a successful upload so that an editor can easily navigate there to work on the rest of the metadata.